README.md

```diff
@@ -84,6 +85,10 @@ The following CMake options are available:
   * `OFF` (default)
   * `ON` Enable IPv6 tests. Requires that IPv6 is
     [setup](https://docs.docker.com/config/daemon/ipv6/) in Docker.
+* `USE_SANITIZER`
+   Compile with a sanitizer. Options depends on
+   [compiler](https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html#index-fsanitize_003daddress),
+   but usually: `address`, `thread`, `undefined`, `leak`
```

Comment:
The format of the other options, with OFF (default) and ON [description] bullets?

I don't see where this option is handled. Is it a CMake thing or a GCC only option? Perhaps we can add something more generic to allow adding any custom GCC flags?

Reply:
CMake's `options` are ON/OFF only, but are configured with a -D, just like how a CMake variable can be set using -D
So when needing someing else then on/off a varaible can be used, but its a bit harder to see then a `option`